addons/im_livechat/static/src/core/common/livechat_command_dialog.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.livechat_command_dialog">
+        <ActionPanel title="props.title" icon="props.icon" resizable="false">
+            <div class="input-group my-2">
+                <div class="o-livechat-LivechatCommandDialog-form form-control bg-view p-0" aria-autocomplete="list">
+                    <input type="text" class="border-0 h-100 rounded px-2" accesskey="Q" t-att-placeholder="props.placeholderText" t-on-keydown="onKeydownCreate" t-ref="inputRef" t-model="state.inputText"/>
+                </div>
+                <button class="btn btn-primary ms-1 rounded" t-on-click="executeCommand" t-att-disabled="!state.inputText"><t t-out="props.title"/></button>
```

Comment:
- Can `t-out` on the `<button>` immediately, rather than put a child `<t>` node. Can use `<button />` syntax thanks to no having to define any child.
- Would remove the `.rounded` on button: the input is rounded on left side but not right side, feels best to have same with the button from the `.input-group`.